irods/test/connection_test.py

```diff
@@ -41,12 +41,10 @@ def test_failed_connection(self):
         # set port back
         self.sess.pool.account.port = saved_port
 
-    def test_send_failure(self):
+    def test_multiple_disconnect(self):
         with self.sess.pool.get_connection() as conn:
-            # try to close connection twice, 2nd one should fail
+            # disconnect() may now be called multiple times without error.
             conn.disconnect()
-            with self.assertRaises(NetworkException):
-                conn.disconnect()
```

Comment:
Shouldn't we leave the second disconnect? Or is the context manager going to call disconnect the second time? Maybe saying so in a comment would be helpful for people wondering in the future :)

Reply:
Yes it's the context manager doing it the second time. Will comment this.